cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/more/index.js

```diff
@@ -14,47 +14,38 @@ function More() {
   const history = useHistory();
   const [clearDataModalOpen, toggleClearDataModal] = useToggle(false);
 
-  const clearAllData = useCallback(async (evt) => {
-    evt.preventDefault();
-    await eventStore.clearAllData();
-    history.push('/');
-  }, [eventStore, history]);
+  const clearAllData = useCallback(
```

Comment:
This is knit picky, but I would recommend reverting this back to how it was before.

Breaking a function to a new line for the arguments tends to not fall in line with JS formatting/syntax. It doesn't break anything necessarily but maintains consistency in code style.

Also, I have updated this function for my issue, and this unnecessary change will cause conflicts with that PR.

Reply:
@jbevis I am not sure what you mean.  I did not change this.   Could we discuss?